kimchi/src/circuits/expr.rs

```diff
@@ -368,7 +368,7 @@ pub enum Expr<C> {
     VanishesOnLast4Rows,
     /// UnnormalizedLagrangeBasis(i) is
     /// (x^n - 1) / (x - omega^i)
-    UnnormalizedLagrangeBasis(usize),
+    UnnormalizedLagrangeBasis(i32),
```

Comment:
Not `u32`?

Reply:
That's what lets this PR work; otherwise, we have to fix the domain size.